lib/perl/Genome/Model/Build/GenePrediction/Bacterial.pm

```diff
@@ -172,18 +172,19 @@ sub regex_files_for_diff {
     );
 }
 
-sub regex_for_custom_diff {
+sub special_compare_functions {
     return (
-        ace         => '\.ace$',
-        ks_bz2      => '\.ks\.bz2$',
-        sorted_bz2  => '\.sorted\.bz2$',
-        fasta       => '\.fasta$',
-        sqlite      => 'sqlite-[^-]+-\d{4}_\d{2}_\d{2}.dat$',
-    ), $_[0]->SUPER::regex_for_custom_diff(@_);
+        qr(\.ace$) => sub {my ($a, $b) = @_; !Genome::Model::Build::GenePrediction::Bacterial::diff_ace($a, $b);},
```

Comment:
It might be simpler if each of these passed `@_` along instead of declaring `$a` and `$b` (which are also magic variables for sorting).